src/components/Trip/ActivityPopup/ActivityPopup.css

```diff
@@ -0,0 +1,19 @@
+.tripScreen .popup-back {
```

Comment:
in order to keep things consistent, in the future I think we can instead use MUI's inline styling, so that we won't have separate .css files only for some components. examples: https://material-ui.com/customization/components/#shorthand